platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -386,6 +386,11 @@
         this.emergencyRecoveryManager = Objects.requireNonNull(emergencyRecoveryManager, "emergencyRecoveryManager");
         final Time time = Time.getCurrent();
 
+        // When the setting goes away, this method call should be deleted
```

Comment:
Consider annotation with `Future work:`

Reply:
This code was removed since the static initialization is no longer happening.